dv/uvm/core_ibex/scripts/compile-tb.py

```diff
@@ -35,6 +37,32 @@ def main() -> int:
 
     os.makedirs(args.output, exist_ok=True)
 
+    # Setup environment variables for the tb to link against the compiled ISS
+    if args.en_cosim:
+        _spike_iss_pc = 'riscv-riscv riscv-disasm riscv-fdt'
+
+        if not subprocess.run(shlex.split(f'pkg-config --exists {_spike_iss_pc}'),
+                              check=True):
+            raise EnvironmentError(f'Failed to find "{_spike_iss_pc}" pkg-config\
+            packages. Did you set the PKG_CONFIG_PATH correctly?')
+
+        def _get_iss_pkgconfig_flags(cmd, iss_pc):
```

Comment:
I don't think this is closing over anything? If not, I'd probably suggest moving it out of the parent function to keep everything reasonably short.

Reply:
This is the only comment I didn't fully address in the new changes, since it does close over args.simulator, and I like the readability of keeping the function local to its use. But if you feel strongly I don't mind moving it either.

Reply:
Ah, I see. I'd probably prefer moving it out: that way, it's more explicit what we're passing to the helper function (and it's `args.simulator`, instead of `args`) and it also avoids the surrounding function getting too enormous.